Gems/ScriptCanvas/Code/Include/ScriptCanvas/Libraries/Operators/Math/OperatorDiv.cpp

```diff
@@ -103,6 +103,30 @@ namespace ScriptCanvas
                 {}
             };
 
+            template <>
+            struct OperatorDivImpl<Data::VectorNType>
+            {
+                OperatorDivImpl(Node* node)
+                    : m_node(node)
+                {}
+
+                Data::VectorNType operator()(const Data::VectorNType& a, const Datum& b)
+                {
+                    const Data::VectorNType* divisor = b.GetAs<Data::VectorNType>();
+                    if (divisor->IsZero())
```

Comment:
This should also check that the divisor isn't nullptr either and probably return 0 in that case as well.

Reply:
Yup, sounds good 👍